src/promptflow-tools/tests/test_handle_openai_error.py

```diff
@@ -342,3 +342,47 @@ def test_aoai_with_vision_model_extra_fields_error(self, azure_open_ai_connectio
 
         assert "extra fields not permitted" in exc_info.value.message
         assert "Please kindly avoid using vision model in LLM tool" in exc_info.value.message
+
+    def test_chat_prompt_with_invalid_assistant_message(self, azure_open_ai_connection):
```

Comment:
The two tests can also be merged to one by using parametrize.

Reply:
fixed